deep4cast/forecasters.py

```diff
@@ -163,6 +163,39 @@ def _check_is_standardized(self):
         if not self._is_standardized:
             raise ValueError('The data has not been standardized.')
 
+    def predict_samples(self, ts, n_sample=1000, quantiles=(0.025,0.975)):
```

Comment:
There is a lot of code duplication with predict_point_estimate. We should avoid this and instead have either one method or one method calls the other.